riscv/tests/riscv.rs

```diff
@@ -420,13 +428,15 @@ fn verify_riscv_crate_with_data<S: serde::Serialize + Send + Sync + 'static>(
     runtime: &Runtime,
     data: Vec<(u32, S)>,
 ) {
-    verify_riscv_crate_from_both_paths(case, inputs, runtime, Some(data))
+    verify_riscv_crate_impl(case, inputs, runtime, true, true, Some(data))
 }
 
-fn verify_riscv_crate_from_both_paths<S: serde::Serialize + Send + Sync + 'static>(
+fn verify_riscv_crate_impl<S: serde::Serialize + Send + Sync + 'static>(
     case: &str,
     inputs: Vec<GoldilocksField>,
     runtime: &Runtime,
+    via_elf: bool,
+    via_asm: bool,
```

Comment:
Boolean parameters are discouraged, an enum might be clearer.

Reply:
Enum of one, another or both? The values are independent, I think bool is better here.

Reply:
yea ok